src/main/java/io/cdap/delta/datastream/DatastreamEventConsumer.java

```diff
@@ -252,6 +258,26 @@ private void fetchNext() throws Exception {
     }
   }
 
+  private List<SortKey> getSortKeys(GenericRecord record) {
+    GenericArray<?> keys = (GenericArray<?>) record.get(SORT_KEYS_FIELD_NAME);
```

Comment:
nit: addm some comments about what those sort keys are (including the data type)

Reply:
Done

Reply:
I meant adding comments for what sortkeys are returned by datastream plugin. Because others may be curious , why you are only supporting below data types.

Reply:
Updated

Reply:
I guess you didn't push the new commit :)

Reply:
what's Integer for ?
could you list what exact sort keys data stream is returning ?
e.g.
scn : Long

Reply:
Added, Integer was added just for resiliency since Long/Integer are quite similar.